de/nmichael/efa/data/storage/IDataAccess.java

```diff
@@ -483,55 +484,55 @@
      */
-    public DataRecord[] getValidAny(DataKey key) throws EfaException;
+    DataRecord[] getValidAny(DataKey key) throws EfaException;
 
     /**
      * Retrieves an existing data record valid at a specified point in time from this storage object.
      * @param key the key of the data record to retrieve (with or without the validity information)
      * @param t the time at which this record is valid
-     * @throws Exception if the data record does not exist, is locked or the operation fails for another reason
+     * @throws EfaException if the data record does not exist, is locked or the operation fails for another reason
      */
-    public DataRecord getValidAt(DataKey key, long t) throws EfaException;
+    DataRecord getValidAt(DataKey key, long t) throws EfaException;
 
     /**
      * Retrieves the latest valid version of an existing data record from this storage object.
      * @param key the key of the data record to retrieve (with or without the validity information)
-     * @throws Exception if the data record does not exist, is locked or the operation fails for another reason
+     * @throws EfaException if the data record does not exist, is locked or the operation fails for another reason
      */
-    public DataRecord getValidLatest(DataKey key) throws EfaException;
+    DataRecord getValidLatest(DataKey key) throws EfaException;
 
     /**
      * Retrieves the nearest valid version of an existing data record from this storage object.
      * @param key the key of the data record to retrieve (with or without the validity information)
      * @param earliestValidAt earlist timestamp which is accepted to fall into the validity range of the record
      * @param latestValidAt latest timestamp which is accepted to fall into the validity range of the record
      * @param preferredValidAt preferred validity shall be as close as possible to this timestamp
-     * @throws Exception if the data record does not exist, is locked or the operation fails for another reason
+     * @throws EfaException if the data record does not exist, is locked or the operation fails for another reason
      */
-    public DataRecord getValidNearest(DataKey key, long earliestValidAt, long latestValidAt, long preferredValidAt) throws EfaException;
+    DataRecord getValidNearest(DataKey key, long earliestValidAt, long latestValidAt, long preferredValidAt) throws EfaException;
 
     /**
      * Returns true if there are any data records valid at any point in time from this storage object.
      * @param key the key of the data record to retrieve (with or without the validity information)
-     * @throws Exception if the data record is locked or the operation fails for another reason
+     * @throws EfaException if the data record is locked or the operation fails for another reason
      */
-    public boolean isValidAny(DataKey key) throws EfaException;
+    boolean isValidAny(DataKey key) throws EfaException;
```

Comment:
@tfyh There's lots of changes with this pattern. I agree that maybe "public" isn't necessary here, and the javadoc comment should say EfaException. Could it be that your Idea IDE automatically reformats classes you touch? Is this something you can disable? While some of these changes may be good, they have nothing to do with the efaCloud changes and make it hard to find the real changes when reviewing the code.